src/components/search/partials/SearchForm/styles.ts

```diff
@@ -0,0 +1,142 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+const searchContainer = css`
+    display: flex;
+    grid-area: header;
+`;
+
+const searchSubmitBtn = css`
+    margin-left: 16px;
+`;
+
+const searchFormGroup = css`
+    display: flex;
+    width: 100%;
+`;
+
+const searchBlockWrapper = css`
+    width: 100%;
+`;
+
+const autosuggestContainer = (theme: GrafanaTheme) =>
+    css`
+    position: relative;
+    background-image: url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="${encodeURIComponent(
+        theme.colors.textWeak
+    )}"><path d="M21.71,20.29,18,16.61A9,9,0,1,0,16.61,18l3.68,3.68a1,1,0,0,0,1.42,0A1,1,0,0,0,21.71,20.29ZM11,18a7,7,0,1,1,7-7A7,7,0,0,1,11,18Z"></path></svg>');
```

Comment:
Is this search icon under an open source license which we can embed?

Reply:
Its path for "search" icon found in Grafana Storybook collection.

Reply:
Can you refactor it similar to https://github.com/grafana/grafana/blob/03d97788ab8f6524e64ad767bea57aeebf59cf13/public/app/core/components/FilterInput/FilterInput.tsx#L17, so that we don't have blobs of licenses code in our sources?
I took a quick look at react-autosuggest and couldn't find a prefix attribute, but it should be possible with a few lines of CSS (see https://github.com/grafana/grafana/blob/03d97788ab8f6524e64ad767bea57aeebf59cf13/packages/grafana-ui/src/components/Input/Input.tsx#L234, you can also re-use Grafana's css by importing `getInputStyles` which is in the @grafana/ui package)

Reply:
Addressed by c31f123